Modules/_collectionsmodule.c

```diff
@@ -2509,17 +2514,12 @@ PyDoc_STRVAR(module_doc,
 - defaultdict:  dict subclass with a default value factory\n\
 ");
 
-static struct PyMethodDef module_functions[] = {
-    {"_count_elements", _count_elements,    METH_VARARGS,   _count_elements_doc},
-    {NULL,       NULL}          /* sentinel */
-};
-
 static struct PyModuleDef _collectionsmodule = {
     PyModuleDef_HEAD_INIT,
     "_collections",
     module_doc,
     -1,
-    module_functions,
```

Comment:
This needs to be kept in order for the function to be exported in the module.

Reply:
Right.